static/js/containers/pages/b2b/B2BReceiptPage.js

```diff
@@ -113,6 +113,9 @@ export class B2BReceiptPage extends React.Component<Props, State> {
 
     const totalPrice = new Decimal(orderStatus.total_price)
     const itemPrice = new Decimal(orderStatus.item_price)
+    const discount =
+      orderStatus.discount !== null ? new Decimal(orderStatus.discount) : null
```

Comment:
`discount` is defined in `B2BOrderStatus` as `?string`, so couldn't it be `undefined` as well? If so, we should change the type definition or this value check